pkgs/development/python-modules/pycryptodome/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl, python, buildPythonPackage, gmp }:
+
+buildPythonPackage rec {
```

Comment:
no, it's good to leave it here, since `python-packages.nix` is getting way too big and having a function is more flexible as well.